CONTRIBUTING.md

```diff
@@ -227,258 +227,11 @@ When the script has finished running, it will print instructions telling you wha
 If it has been a while since you set up the virtualenv, make sure you have
 the latest mypy (`pip install -r requirements-tests.txt`) before running the script.
 
-### Supported type system features
-
-Since PEP 484 was accepted, there have been many other PEPs that added
-new features to the Python type system. In general, new features can
-be used in typeshed as soon as the PEP has been accepted and implemented
-and most type checkers support the new feature.
-
-Accepted features that *cannot* yet be used in typeshed include:
```

Comment:
I think this list is still useful, and it makes sense to keep it in typeshed because it's specifically about this repo's policies.

Reply:
Whoops, I kind of forgot about this PR.

Maybe it's best to link to https://github.com/python/typeshed/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-tracker, instead of listing the missing features? The issues are more likely to be kept up-to-date than the list in CONTRIBUTING.

Reply:
Sure, sounds good

Reply:
We could also consider using a meta-issue tracker to track the issue trackers. The advantage of using an issue to track the issue trackers is that items will automatically be ticked off as completed if issue trackers on the bullet-pointed list are ticked off. We could link to the meta-issue tracker from `CONTRIBUTING.md` and pin it on the issue tracker.